docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,23 @@ NULL
 
 -->
 
+## Version 4.3.1
+
+Date: May 15, 2024
+
+### Improvements
+
+- Implemented a new billing system for CronJobs and Jobs that is more favorable to customers than existing container billing.
+- Enhanced the reliability of the Kubernetes initialization structure.
+- Corrected inaccurate information about `agent_started` events in Sematext Cloud that gave the impression the agent was frequently restarting.
+
+### Bug Fixes
+
+- Resolved a service discovery bug occurring with ElasticSearch version 8.
+- Fixed a bug that affected reading Nginx configuration files, to able to discover your `server_status` path.
```

Comment:
to able to discover your server_status path ->  to enable discovery of your `server_status` path.